src/tabs/home/Features/index.js

```diff
@@ -3,142 +3,226 @@ import { Query } from 'react-apollo';
 import { get } from 'lodash';
 // import PropTypes from 'prop-types';
 
-import { styled, ActionListCard, H3, H6 } from '@apollosproject/ui-kit';
 import { AnalyticsConsumer } from '@apollosproject/ui-analytics';
+import ActionListFeature from './ActionListFeature';
+import CampaignItemListFeature from './CampaignItemListFeature';
+import HorizontalCardListFeature from './HorizontalCardListFeature';
+import VerticalCardListFeature from './VerticalCardListFeature';
 
 import GET_FEED_FEATURES from './getFeedFeatures';
 
-const StyledH6 = styled(({ theme }) => ({
-  color: theme.colors.text.tertiary,
-}))(H6);
+const handleOnPressActionItem = ({
+  action,
+  navigation,
+  relatedNode,
+  track,
+}) => {
+  if (action === 'READ_CONTENT') {
+    navigation.navigate('ContentSingle', {
+      itemId: relatedNode.id,
+      transitionKey: 2,
+    });
+  }
+  if (action === 'READ_EVENT') {
+    navigation.navigate('Event', {
+      eventId: relatedNode.id,
+      transitionKey: 2,
+    });
+  }
+  track({
+    eventName: `Clicked Home Feed Feature`,
+    properties: [
+      {
+        field: 'Node ID',
+        value: relatedNode.id,
+      },
+    ],
+  });
+};
 
-// const handleOnPressActionItem = (id) =>
-//   this.props.navigation.navigate('ContentSingle', {
+// const handleOnPressCardActionButton = ({ id, navigation, title }) =>
```

Comment:
should we just delete these commented out lines?

Reply:
We can if you think it would be better. These were in their file so I left them. But I'd have no problem removing them. Whatcha think?

Reply:
i'd say delete em